### PR TITLE
Providing Implementation for Spanning Tree for Dell Switches

### PR DESCRIPTION
**Pull request by idjaw**

### CODE REVIEW

**Comment by lindycoder** [comment]
:+1:

File: netman/adapters/switches/dell.py [hunk 1]
```diff
@@ -215,6 +215,16 @@ def parse_interface_names(self, status_list):
 
         return interfaces
 
+    def enable_interface_spanning_tree(self, interface_id):
+        with self.config(), self.interface(interface_id):
+            self.ssh.do("spanning-tree disable")
```
Comment (mat128) [inline netman/adapters/switches/dell.py]: That disables spanning tree

File: netman/adapters/switches/dell.py [hunk 2]
```diff
@@ -215,6 +215,16 @@ def parse_interface_names(self, status_list):
...
+            self.ssh.do("spanning-tree portfast")
+
+    def disable_interface_spanning_tree(self, interface_id):
+        with self.config(), self.interface(interface_id):
+            self.ssh.do("no spanning-tree disable")
```
Comment (mat128) [inline netman/adapters/switches/dell.py]: That enables spanning tree on Dell.

> **spanning-tree disable**
> Use the spanning-tree disable command in Interface Configuration mode to
> disable spanning-tree on a specific port. To enable spanning-tree on a port,
> use the no form of this command.

**Comment by idjaw** [comment]
Closing Pull-Request to discuss STP and determine final implementation details.

**Pull request closed without merging**